include/mbedtls/ssl.h

```diff
@@ -324,6 +329,9 @@
 #define MBEDTLS_SSL_SESSION_TICKETS_DISABLED     0
 #define MBEDTLS_SSL_SESSION_TICKETS_ENABLED      1
 
+#define MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_DISABLED  0
+#define MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_ENABLED   1
```

Comment:
This looks like a boolean setting. Why are we defining aliases for true and false? Are we going to add `MBEDTLS_SSL_ENABLE_NEW_SESSION_TICKETS_FILE_NOT_FOUND` later?

Reply:
I've just followed what we have for `mbedtls_ssl_conf_session_tickets` and the definition of `MBEDTLS_SSL_SESSION_TICKETS_DISABLED` and `MBEDTLS_SSL_SESSION_TICKETS_ENABLED`.

Reply:
At this point we should emulate what other similar settings are doing, which is have the enabled/disabled macros.

(In the future, I'd be happy to `#include <stdbool.h>` and remove all these aliases for true and false.)